draft-meynell-panrg-scion-research-questions.md

```diff
@@ -276,6 +276,18 @@ When a client contacts a server, it is usually understood that it wants the serv
 
 There are some nuances: Usually the server's API will store the initial address of the client to be used through all the session. We might need to take this into account.
 
+A related question: how long before expiration should we still use a path? How do we handle that?
+
+Do we actually need to solve this reverse path refresh problem?
+
+* CONTRA: It is probably rare that a server needs to send data for a long time without the client ever answering back.
+* PRO: The client may happen to have an old-ish path. If we can't refresh, the client always needs to consider whether a path is valid "long enough", which might only be possible to guess.
+* CONTRA: Sending keep-alives sounds like a connection based protocol. It alo means we need to figure out when to stop sending keep alives.
+* CONTRA: It may be better to solve this in the application layer or in the overlay protocol, where we we know more about
```

Comment:
Correct, we talked about moving this to 7.  Implications of Path Awareness for the Transport and Application Layers.
What do you think?

Reply:
Do you mean moving the whole 4.2 into 7? We could do that, but I think the application layer is only a _possible_ _solution_, while the _problem_ is actually on the data plane?

Also, in case that was the suggestion, I would rather not split it up, because this really seems to be a single topic, or not?

What do you think?

Reply:
I see your point, and I agree this is a data-plane issue.
At least the statement of the problem should remain under data-plane. We can make it clear in the solution that it belongs to the transport and/or application layer (we can actually provide both solutions).
I will later open a PR tackling other changes in the expired reverse path server issue.

Reply:
+1 for moving this into 7 Implications of Path Awareness for the Transport and Application Layers

Maybe MP-QUIC has something about refreshing paths?